src/plugins/template/backend/int_executable.cpp

```diff
@@ -83,14 +83,20 @@ class TemporaryOverrideOutputs {
         for (size_t i = 0; i < args.size(); ++i) {
```

Comment:
@zhaixuejun1993 Please try to apply `TemporaryOverrideOutputs` only for parameters before the node cycle in the call method. I hope it should help

Reply:
@ilyachur I had reproduced the CI tests failed in local, the error caused by the changes in class `TemporaryOverrideOutputs`, try u suggestions but doesn't work, maybe I'm not very familiar with the concept of `TemporaryOverrideOutputs` & `Parameter`. **Can u clarify it & show more details about the suggestion change**.
![image](https://user-images.githubusercontent.com/52686861/227783858-d050e655-e529-4c78-ba0e-39c3fdfe75b9.png)